<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ordering</class>
 <widget class="QDialog" name="Ordering">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>112</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>BankGothic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ordering</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptedBtn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>281</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="rejectedBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>281</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Begin Date:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="beginDate">
        <property name="date">
         <date>
          <year>2017</year>
          <month>3</month>
          <day>31</day>
         </date>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>  End Date:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="endDate">
        <property name="date">
         <date>
          <year>2017</year>
          <month>3</month>
          <day>31</day>
         </date>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>  Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>IDcard:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cardEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
